ideas/cool-project.md

```diff
@@ -0,0 +1,19 @@
+# Talk about a Cool Project You Did
+
+|  Author  | Last Updated |       Time Estimate        |
+| :------: | :----------: | :------------------------: |
+| Jason Howe |  09/27/2022  | Short - 🟩🟩🟩⬜️⬜️ - Long |
+
+## Description
+
+Write a cool tool, and give a presentation on it!
```

Comment:
I like the shilling for the presentations at RITSEC, but this idea seems too generic to fit into the ideas list.